dev/data/dashboard.json

```diff
@@ -1911,6 +1911,42 @@
               }
             }
           }
+        },
+        "LineVisualOptions": {
+          "kind": "Panel",
+          "spec": {
+            "display": {
+              "name": "Line Visual Options",
```

Comment:
Maybe call `Custom Visual Options` to match how you're describing? (Time series chart with custom visual options)?